integration/fuse/bin/alluxio-fuse

```diff
@@ -18,21 +18,9 @@ get_env() {
   . ${ALLUXIO_LIBEXEC_DIR}/alluxio-config.sh
 
   ALLUXIO_FUSE_JAR=${SCRIPT_DIR}/../target/alluxio-integration-fuse-${VERSION}-jar-with-dependencies.jar
-  FUSE_MAX_WRITE=131072
```

Comment:
@cheyang  is this used any where?

Reply:
I think it's not used. We can remove it.